.travis.yml

```diff
@@ -0,0 +1,3 @@
+language: python
+install: "pip install -r requirements.txt"
+script: wiki_cat
```

Comment:
Does this mean Travis CI is fully setup. If not maybe this should wait for a different PR.